lib/src/rules/prefer_final_fields.dart

```diff
@@ -190,7 +190,8 @@ class _Visitor extends SimpleAstVisitor<void> {
           if (isFieldInAllConstructors) {
             rule.reportLint(variable);
           }
-        } else if (element.initializer != null) {
+        } else if (element is PropertyInducingElement &&
```

Comment:
This is semantically correct, but a tiny bit confusing the reader.
It is not that there is an *option* that the `element` has any other type.
As such, it might be more explicit to cast it in the `final element` declaration.

Reply:
If that's safe/preferred, I'm all for it and I updated.  Take another look to make sure I got it right?